lib/cusparse/conversions.jl

```diff
@@ -152,20 +208,94 @@ for (cname,rname,elty) in ((:cusparseScsc2dense, :cusparseScsr2dense, :Float32),
         function CUDA.CuMatrix{$elty}(csr::CuSparseMatrixCSR{$elty}; ind::SparseChar='O')
             m,n = csr.dims
             denseA = CUDA.zeros($elty,m,n)
-            lda = max(1,stride(denseA,2))
-            cudesc = CuMatrixDescriptor(CUSPARSE_MATRIX_TYPE_GENERAL, CUSPARSE_FILL_MODE_LOWER, CUSPARSE_DIAG_TYPE_NON_UNIT, ind)
-            $rname(handle(), m, n, cudesc, nonzeros(csr),
-                   csr.rowPtr, csr.colVal, denseA, lda)
-            denseA
+            if version() >= v"11.1"
```

Comment:
CUDA libraries are versioned independently starting with CUDA 11, and `version()` returns that CUSPARSE version. So the version check needs to be more specific probably, see https://docs.nvidia.com/cuda/cuda-toolkit-release-notes/index.html#cuda-major-component-versions

Reply:
Oh shoot, thanks!